tests/e2e/jest.config.json

```diff
@@ -1,8 +1,6 @@
 {
   "preset": "ts-jest",
   "testEnvironment": "node",
-  "globalSetup": "<rootDir>/_setup.ts",
-  "globalTeardown": "<rootDir>/_teardown.ts",
   "setupFilesAfterEnv": ["<rootDir>/_setupLogging.ts"],
-  "testTimeout": 180000
+  "testTimeout": 540000
```

Comment:
do these tests take so long because we use the normal ethereum test network?
in the future we could consider setting up our docker with goerli (as Goth did)..

Reply:
I just doubled these during testing as I came across timeouts during development. Could be solved now that we have the invoice bug fixed. I will revert this